### ISSUE TITLE
 find file references

### ISSUE THREAD

**Issue by reslear** [zed-extensions/vue#3]
<img width="420" src="https://user-images.githubusercontent.com/16279759/187038026-b905c819-447e-40ae-8798-9ac8aac91312.png">

https://github.com/vuejs/language-tools/issues/1724

**Comment by olrtg** [comment]
Yeah this is not possible at the moment it seems. The extensions api doesn't allow us to add commands to the command palette yet.
